src/biome/text/about.py

```diff
@@ -95,9 +95,8 @@ def version_matches(release_branch_version: str, configured_version: str) -> boo
     if release_version and not repo_tag:
         assert version_matches(release_version, version)
         return version
-    today = datetime.today().strftime("%Y%m%d%H%M%S")
 
-    return repo_tag if repo_tag else f"{version}.dev{today}+{commit}"
+    return repo_tag if repo_tag else f"{version}.dev+{commit}"
```

Comment:
Warning: This could bring problems with package updates for dev versions, since pypi repository sort by version name (There is no sorting on commit hash)

Reply:
Was not aware of that. Instead of `datetime.today` i added the **commit date** in the same format as before to the version str, this should solve the problem.